core/.gas-snapshot

```diff
@@ -1,15 +1,15 @@
 BorrowerGasTest:test_addMargin() (gas: 16075)
-BorrowerGasTest:test_borrow() (gas: 97662)
+BorrowerGasTest:test_borrow() (gas: 100023)
```

Comment:
Many of these increased by ~2000 gas, because they represent the case where `IManager.callback` is returning an  empty array -- before we did nothing, now we have to read from storage. `sload` costs 2000 gas.